src/main/resources/templates/docker/docker-compose.yml

```diff
@@ -4,48 +4,48 @@ services:
   config-server:
     image: "demo/cloud-config-server"
     ports:
-      - "{{cloud-config-server-port}}:{{cloud-config-server-port}}"
+      - "{{port}}:{{port}}"
```

Comment:
Does that mean all services are using same `{{port}}`?

Reply:
yes, but there are kinds of different `Map` represent microservices, for each one has entry `port`.